app/components/service/service.html

```diff
@@ -72,6 +72,13 @@
                 <input type="text" class="form-control" ng-model="service.Image" ng-change="updateServiceAttribute(service, 'Image')" ng-disabled="isUpdating" />
               </td>
             </tr>
+            <tr>
```

Comment:
The condition should be located on the `<tr>` element so that we don't display an empty row.

See the screenshots below:

![portainer 12](https://user-images.githubusercontent.com/5485061/27983358-c2fe2f8a-63ba-11e7-8110-325b6d4fd509.png)

![portainer 13](https://user-images.githubusercontent.com/5485061/27983359-c6bdf966-63ba-11e7-83e8-a0b5db99b05b.png)